package-lock.json

```diff
@@ -19292,7 +19292,7 @@
     },
     "react-content-loader": {
       "version": "3.4.2",
-      "resolved": "https://registry.npmjs.org/react-content-loader/-/react-content-loader-3.4.2.tgz",
+      "resolved": "https://repository.engineering.redhat.com/nexus/repository/registry.npmjs.org/react-content-loader/-/react-content-loader-3.4.2.tgz",
```

Comment:
Done! But also why exactly? I thought the idea was to change everything to use repository.engineering.redhat.com

Reply:
We're just supposed to use that for our registry internally. The problem is that if there is an outside contributor, they won't be able to use that since it's Red Hat internal specific.

Reply:
ahh okay